### ISSUE TITLE
Init script case sensitive error problem.

### ISSUE THREAD

**Issue by s4l1h** [kasparsd/php-7-debian#42]
He could not catch the following error.

[20-Jan-2017 16:53:30] NOTICE: PHP message: PHP Fatal error: The fpm-fcgi SAPI is not supported by pthreads in Unknown on line 0 [20-Jan-2017 16:53:30] NOTICE: PHP message: PHP Fatal error: Unable to start pthreads module in Unknown on line 0

Os: Debian 8

**Comment by kasparsd** [comment]
@s4l1h I just did a test and got the following output where the `ERROR` is in uppercase:

```
$ /usr/local/php7/sbin/php7-fpm --fpm-config /usr/local/php7/etc/php-fpm.conf -t
[25-Feb-2017 10:56:31] NOTICE: PHP message: PHP Warning:  PHP Startup: memcached: Unable to initialize module
Module compiled with module API=20151012
PHP    compiled with module API=20160303
These options need to match
 in Unknown on line 0
[25-Feb-2017 10:56:31] ERROR: failed to open error_log (/var/log/php7-fpm.log): Permission denied (13)
[25-Feb-2017 10:56:31] ERROR: failed to post process the configuration
[25-Feb-2017 10:56:31] ERROR: FPM initialization failed
```

Did it actually failed to start the FPM daemon in your case?

**Comment by kasparsd** [comment]
So the current implementation won't even catch that because it looks for `[ERROR]` instead of `ERROR`. Merging your fix. Thank you!

**Comment by kasparsd** [comment]
Fixed with #43.
